lib/inspect_filesize.c

```diff
@@ -125,12 +127,14 @@ static bool filesize_driver(struct rpminspect *ri, rpmfile_entry_t *file)
         params.severity = RESULT_INFO;
         params.verb = VERB_OK;
         params.remedy = NULL;
+        result = true;
```

Comment:
Should this be result or reported? I see:

`bool result = true;`

up on line 35

Reply:
It needs to be result.  reported is set to true after I call add_result()

Reply:
Thanks for the confirmation!